internal/app/orchestrator/orchestrator_test.go

```diff
@@ -285,8 +269,8 @@ func TestCachedResponse(t *testing.T) {
 func TestMultipleWatchersAndUpstreams(t *testing.T) {
 	upstreamResponseChannelLDS := make(chan *v2.DiscoveryResponse)
 	upstreamResponseChannelCDS := make(chan *v2.DiscoveryResponse)
+	mapper := mapper.NewMapper(t)
```

Comment:
What do you think of calling these mock instantiators `NewMock`? Reading the test code it's not immediately apparent that they're mocks and how they differ from `mapper.New` unless the package is deliberately named `mock`.

Reply:
I agree `NewMock` is clearer